docs/man/3.x/index.html

```diff
@@ -3815,7 +3815,7 @@ <h3 id="_registering_subcommands_declaratively">13.2. Registering Subcommands De
 <div class="sect2">
 <h3 id="_subcommand_aliases">13.3. Subcommand Aliases</h3>
 <div class="paragraph">
-<p>Commands may optionally define an <code>aliases</code> attribute to provide alternate names for commands that will be recognized by the parser. Aliases are displayed in the default help output. For example:</p>
+<p>Commands may optionally define an <code>aliases</code> attribute to provide alternate names for subcommands that will be recognized by the parser. Aliases are displayed in the default help output. For example:</p>
```

Comment:
Let's leave out the "for commands" and "for subcommands" altogether:

> ... provide alternate names that will be recognized by the parser.

Reply:
👍